wagtail/admin/templates/wagtailadmin/shared/chooser_breadcrumb.html

```diff
@@ -1,33 +1,24 @@
 {% load i18n wagtailadmin_tags %}
 
```

Comment:
Note.we should pull this content 'up' into the parent template. This is because the locale selector is not part of the breadcrumbs anymore and there are only two lines of code for the actual breadcrumbs. 

I'll try to do this when I merge this in.